sql/mob_skill_lists.sql

```diff
@@ -3997,7 +3997,14 @@ INSERT INTO `mob_skill_lists` VALUES ('JailerOfTemperance',4047,1467);
 INSERT INTO `mob_skill_lists` VALUES ('JailerOfTemperance',4047,1468);
 INSERT INTO `mob_skill_lists` VALUES ('JailerOfTemperance',4047,1469);
 
--- 4048 Available
+-- Ouryu from Ouryu Cometh
+INSERT INTO `mob_skill_lists` VALUES ('OuryuComethWyrm',4048,1299);
+INSERT INTO `mob_skill_lists` VALUES ('OuryuComethWyrm',4048,1300);
+INSERT INTO `mob_skill_lists` VALUES ('OuryuComethWyrm',4048,1301);
+INSERT INTO `mob_skill_lists` VALUES ('OuryuComethWyrm',4048,1303);
+INSERT INTO `mob_skill_lists` VALUES ('OuryuComethWyrm',4048,1304);
+INSERT INTO `mob_skill_lists` VALUES ('OuryuComethWyrm',4048,1412); -- AoE version of Absolute Terror
+INSERT INTO `mob_skill_lists` VALUES ('OuryuComethWyrm',4048,1306);
```

Comment:
Move to the bottom of the list with a new ID. See above.

Reply:
I moved the Ouryu change to a new PR dealing with several mobskills, will be easier to modularize several there.